views/members.pug

```diff
@@ -0,0 +1,16 @@
+doctype html
+html(lang="en")
+    head
+        include head
+        link(href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet")
+        title Members
+    body
+        h1 Fitness buddy
+        img.logo-image(src="img/dumbbell.png" alt="fitness-buddy-logo")
+        #container-margin
+            a.navbar-brand(href="/logout") Logout
+            h2 Welcome
+              span.member-name
+        br
+        script(src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js")
+        script(type="text/javascript" src="js/members.js")
```

Comment:
Is this now deprecated?

Reply:
Yes. I've now moved this to a 'to-be-deleted' folder, just in case we wanted to restore this page